ui/components/multichain/account-list-menu/account-list-menu.tsx

```diff
@@ -225,6 +226,11 @@ export const AccountListMenu = ({
   ///: BEGIN:ONLY_INCLUDE_IF(build-flask)
   const { pathname } = useLocation();
   ///: END:ONLY_INCLUDE_IF
+  const [searchQuery, setSearchQuery] = useState('');
+  const [actionMode, setActionMode] = useState(ACTION_MODES.LIST);
+  ///: BEGIN:ONLY_INCLUDE_IF(solana,build-flask)
+  const [primaryKeyring] = useSelector(getMetaMaskKeyrings);
+  ///: END:ONLY_INCLUDE_IF
```

Comment:
Dropping `entropySource` support for Bitcoin for now.

```suggestion
  ///: BEGIN:ONLY_INCLUDE_IF(solana)
  const [primaryKeyring] = useSelector(getMetaMaskKeyrings);
  ///: END:ONLY_INCLUDE_IF
```